src/test/java/org/numenta/nupic/network/RegionTest.java

```diff
@@ -405,7 +409,7 @@ public void testIsLearn() {
         Network n = Network.create("test network", p)
             .add(Network.createRegion("r1")
                 .add(Network.createLayer("1", p)
-                    .alterParameter(KEY.AUTO_CLASSIFY, Boolean.TRUE))
+                    .alterParameter(KEY.AUTO_CLASSIFY, false))
```

Comment:
@Hopding Don't understand why this is false if you're setting the inferred fields below? Also, shouldn't this be true anyway to keep the test like it is?